app/javascript/src/PaymentGateways/stripe/styles/stripe.scss

```diff
@@ -157,3 +157,28 @@ $stripe-form-spinner-color: #5469d4;
     }
   }
 }
+
+#stripe-form {
```

Comment:
There is `.stripe-form` class definition in this file, but it doesn't seem to be used...
I'm not sure though whether to remove it in this PR.

Reply:
If it's not used, we should remove it :v:

Reply:
Thanks @josemigallas ! I was not aware of that page, and in fact, I am not sure whether it is used at all :thinking: 
But I'll review this.